### PR TITLE
[editor-js] Implement underline functionality

### PR DESCRIPTION
**Pull request by fajardm**
### Summary
Implement underline functionality

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    }
+                ]
+            }
+        ]
+    },
+    output: {
+        path: __dirname + '/dist',
+        publicPath: '/',
+        filename: 'bundle.js',
+        library: 'Paragraph',
```
Comment (alcarazzam) [inline webpack.config.js]: Should this be `Underline`?

Comment (fajardm) [reply webpack.config.js]: Updated

Comment (fajardm) [reply webpack.config.js]: > Should this be `Underline`?

Yes, updated

**Comment by vincentdesmares** [comment]
@m-alzam I think this requires a merge :) The repository would be IMO better with those changes than empty.

**Comment by alcarazzam** [comment]
> @m-alzam I think this requires a merge :) The repository would be IMO better with those changes than empty.

Yes, but I'm not the owner of the repo :)

**Comment by neSpecc** [comment]
Absolutely. We will check it soon.

**Review by neSpecc — APPROVED** [review]

**Pull request merged**